src/components/OSCALControlProse.js

```diff
@@ -259,27 +335,128 @@ export function OSCALReplacedProseWithParameterLabel(props) {
     );
   }
 
+  const partialRestData = props.implementedRequirement
+    ? deepClone(props.implementedRequirement)
+    : null;
+  const statement =
+    partialRestData?.statements?.find(
+      (element) => element["statement-id"] === props.statementId
+    ) || null;
+
+  let labelWithProse = props.prose;
+  if (props.label) {
+    labelWithProse = props.label.concat(` ${props.prose}`);
+  }
+
+  const paramId = props.prose.split(
+    RegExp(prosePlaceholderRegexpString, "g")
+  )[1];
+
   return (
     <Typography className={props.className}>
-      {props.label}
-      {props.prose
+      {labelWithProse
         .split(RegExp(prosePlaceholderRegexpString, "g"))
         .map((segment, index) => {
           if (index % 2 === 0) {
             return getTextSegment(segment, index.toString());
           }
-          return getParameterLabelSegment(
-            props.parameters,
-            segment,
-            props.modificationSetParameters,
-            index.toString()
+
+          return (
+            <Typography display="inline" variant="body2">
```

Comment:
This has decreased the font size for labels.  Are we sure we need to wrap here?

Reply:
We do not, so it is getting removed.